packages/commonwealth/client/scripts/utils/polls.ts

```diff
@@ -0,0 +1,28 @@
+import { PollView } from '@hicommonwealth/schemas';
+import { DEFAULT_POLL_DURATION } from '@hicommonwealth/shared';
+import { z } from 'zod';
+
+export interface ExtendedPoll extends z.infer<typeof PollView> {
+  custom_duration?: string;
+}
+
+export interface LocalPoll {
+  options: Array<string>;
+  prompt: string;
+  community_id?: string;
+  custom_duration?: string;
+  ends_at: Date | undefined;
+  votes: [];
+}
+
+export type SetLocalPolls = (params: LocalPoll[]) => void;
+
+export function parseCustomDuration(customDuration?: string): number | null {
+  console.log('Custom Duration Parsing:', customDuration);
```

Comment:
we dont need this?

Reply:
Are you talking about the log or the function? I removed the log but the function is used in 2 different places.

Reply:
I'm gonna resolve this and merge for now but just drop a comment here if there is still an issue and I'll create a quick follow-up fix PR.